docs/docs/self-host/secure-lightdash-with-https.mdx

```diff
@@ -0,0 +1,39 @@
+# Secure Lightdash with HTTPS
+
+To run Lightdash in production it is highly recommended to secure it with HTTPS. This is usually done using your 
```

Comment:
Fun fact, you can't connect Slack if you don't have an HTTPS callback url (there is a workaround though, that I use on localhost)

Reply:
This makes sense, also for SSO. I'll mention that on the docs for those!